fossasia/static/custom.css

```diff
@@ -2,6 +2,7 @@ body{
   font-family: 'Open Sans', "Helvetica Neue", Helvetica, Arial, sans-serif;
   background: #fff;
   color: #777;
+  margin-bottom: 60px;
```

Comment:
You need to set this to the same height as the footer height. If footer height is 100px, this margin should also be 100px;